src/plasma/plasma_store.cc

```diff
@@ -44,8 +47,24 @@ void *dlmemalign(size_t alignment, size_t bytes);
 void dlfree(void *);
 }
 
+namespace std {
+template <>
+struct hash<UniqueID> {
+  size_t operator()(const UniqueID &unique_id) const {
+    return *reinterpret_cast<const size_t *>(unique_id.id + UNIQUE_ID_SIZE -
+                                             sizeof(size_t));
```

Comment:
I think @atumanov has another hash function which we should look into. I think we should merge it with Richards hash function now which is very reasonable and then we can see if we have a better alternative later.